tools/add_triage_signature.py

```diff
@@ -1378,6 +1378,39 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket(report)
 
 
+class MissingMustGatherLogs(ErrorSignature):
+    """
+    This signature add is added in case must-gather logs weren't collected after the failure of the installation.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            function_impact_label="missing_must-gather_logs",
+            comment_identifying_string="h1. Missing must-gather logs",
+        )
+
+    def _process_ticket(self, url, issue_key):
+        md = get_metadata_json(url)
+        cluster_id = md["cluster"]["id"]
+        should_collect_must_gather_logs = False
+        cluster_hosts = md["cluster"]["hosts"]
+        # We require 1 master with progress set to Done in order to be able to collect must-gather logs
+        for host in cluster_hosts:
+            if host["role"] in ("master", "bootstrap") and host["progress"]["current_stage"] == "Done":
+                should_collect_must_gather_logs = True
+        if should_collect_must_gather_logs:
+            triage_logs_tar = get_triage_logs_tar(triage_url=url, cluster_id=cluster_id)
```

Comment:
not sure how frequent it is, but in case we didn't even gather the outer tar file shouldn't we also mark it as "must-gather is missing"?

Reply:
I'm more concerned about someone running the signature scripts without VPN and accidentally signing all the tickets with this signature.
I think that if there are no logs at all, it's another issue and not what we are looking for in this signature.

Reply:
makes sense